HACKING.md

```diff
@@ -32,3 +32,7 @@ file to generate a package list.
 
 If the repository requires a client tls key/cert you can supply them with
 `--key` and `--cert`.
+
+## Generating Open API
+
+For updating the api.go content to match the current api.yaml, simply run `go generate ./...`
```

Comment:
There's a script, `./tools/prepare-source.sh` which runs that among other things. But it isn't mentioned in this repo :) There are similar scripts in osbuild-composer, images, and weldr-client which is probably why nobody mentioned it here.